Dockerfile

```diff
@@ -18,4 +18,6 @@ RUN if [ "$INSTALL_DEV" = "true" ]; then \
 COPY ./rq_dashboard_fast /app/rq_dashboard_fast
 COPY app.py /app
 
-WORKDIR /app
\ No newline at end of file
+WORKDIR /app
+
+CMD ["python3", "app.py"]
```

Comment:
I did not find a way to launch docker image without that line, user has either to have this in Dockerfile here or in his own `docker-compose` - and that contradicts with README.md a bit because there was no `command: ["python3", "app.py"]` there.